examples/expo-example/package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "@knocklabs/react-native-example",
-  "version": "1.3.1",
+  "name": "@knocklabs/expo-example",
+  "version": "0.0.0",
```

Comment:
Given the name and directory of this existing example app is changing, I feel we should restart versioning at `0.0.0`.

Not that it matters much; these example apps are not published anywhere.